docs/start/framework/react/build-from-scratch.md

```
@@ -85,7 +91,11 @@ npm i -D typescript @types/react @types/react-dom @types/node
 
 ## Update Configuration Files
 
-We'll then update our `package.json` to use Vite's CLI and set `"type": "module"`:
+We'll then update our `package.json` to use your bundler's CLI and set `"type": "module"`:
+
+<!-- ::start:tabs variant="bundler" -->
+
+# Vite
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)